roles/kubernetes/kubeadm/tasks/main.yml

```diff
@@ -26,12 +26,6 @@
   run_once: true
   register: temp_token
   delegate_to: "{{ groups['kube-master'][0] }}"
-  when: kubeadm_token is not defined
-
-- name: Set kubeadm_token to generated token
-  set_fact:
-    kubeadm_token: "{{ temp_token.stdout }}"
-  when: kubeadm_token is not defined
```

Comment:
Could this be changed to `temp_token.stdout is defined`?

Reply:
This task is fine. The problem is the previous task for setting temp_token is being skipped.

Reply:
Yeah, I think its because of the run_ounce in the previous task.  Seems to work when that line is commented out.